portal/factories/app.py

```diff
@@ -229,14 +230,36 @@ def configure_logging(app):  # pragma: no cover
         sql_log.setLevel(logging.INFO)
         sql_log.addHandler(sql_file_handler)
 
-    if app.testing or not app.config.get('LOG_FOLDER', None):
+    level = getattr(logging, app.config['LOG_LEVEL'].upper())
+    from ..tasks import logger as task_logger
+    task_logger.setLevel(level)
+    app.logger.setLevel(level)
+
+    # Configure Error Emails for high level log messages, only in prod mode
+    if not (app.debug or app.testing or not app.config.get('ERROR_SENDTO_EMAIL')):
+        ADMINS = app.config['ERROR_SENDTO_EMAIL']
```

Comment:
are you sure we want to disable on `app.debug`?  isn't this how we run stg and whatnot - I feel like those early error emails catch a LOT of bugs...

Reply:
@pbugni I think `app.debug` is only `True` when running the [flask dev server](http://flask.pocoo.org/docs/1.0/quickstart/#debug-mode), at least it's `False` when I checked `flask shell` on stg.us